packages/esm-patient-search-app/src/patient-search.resource.tsx

```diff
@@ -138,20 +139,18 @@ export function useRESTPatients(
   return results;
 }
 
-export function updateRecentlyViewedPatients(patientUuid: string, user: LoggedInUser) {
+export function updateRecentlyViewedPatients(patientUuid: string, user: LoggedInUser, userProperties) {
```

Comment:
It would be better to make this function an export of the `useRecentlyViewedPatients()` hook, e.g., `addPatient()` or something. Then it can use the same session data and we don't need the three-step thing of ensuring that the consumer of this hook also grabs the `userProperties` export and passes it as an argument to `updateRecentlyViewedPatients()`